docs.openc3.com/docs/configuration/interfaces.md

```diff
@@ -80,23 +80,23 @@ INTERFACE INTERFACE_NAME tcpip_client_interface.rb host.docker.internal 8080 808
 </TabItem>
 </Tabs>
 
-| Option              | Description                                                                                                                                                                                                                                                      |
-| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| OPTIMIZE_THROUGHPUT | Number of seconds to wait before writing packets to Redis. By default packets are immediately written to Redis (both the Current Value Table (CVT) and the stream). If you have high rate data this can improve performance by buffering packets before writing. |
+| Option          | Description                                                                                                                                                                                                                                                                                                                                                          |
+| --------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| UPDATE_INTERVAL | Number of seconds to wait before writing packets to Redis. By default packets are written to Redis every second (both the Current Value Table (CVT) and the stream). If you have high rate data and you want faster updates you can lower this value. Note that this will increase processor utilization! This option was previously known as `OPTIMIZE_THROUGHPUT`. |
```

Comment:
every second isn't true.
This is something I want to change for COSMOS 7 but right now updates are immediate.

Reply:
No.  By default we don't use StoreQueued at all (for topics).  Has to be enabled with this setting.
I'm thinking for COSMOS 7 we can default to 0.1 seconds.

Reply:
I didn't see the `@queued = false` as the default. Will update and rebase onto cosmos 7 branch.